src/main/java/mate/academy/library/controllers/BookController.java

```diff
@@ -22,24 +25,24 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
-@Tag(name = "Books management", description = "Endpoint for mapping books")
 @RestController
 @RequiredArgsConstructor
 @RequestMapping(value = "/books")
+@Tag(name = "Books management", description = "Endpoint for mapping books")
 public class BookController {
     private final BookService bookService;
 
-    @PreAuthorize("hasRole('ROLE_USER')")
+    @PreAuthorize("hasRole('ROLE_USER') OR hasRole('ROLE_ADMIN')")

```

Comment:
FYI just removing this line would have the same effect, since they are secured in SecurityConfig